index.html

```diff
@@ -8,9 +8,9 @@
 
```

Comment:
Please add a readme file to your repository 🙏. Make sure to use the microverse readme [template](https://github.com/microverseinc/readme-template) and personalize each section to match your project. Also don't forget to remove all comments. The readme file is very important as it helps to document your project to visitors of your repository